CONTRIBUTING.md

```diff
@@ -24,6 +24,26 @@ requests.
 Major projects are tracked [here](https://github.com/rust-bitcoin/rust-lightning/projects).
 Major milestones are tracked [here](https://github.com/rust-bitcoin/rust-lightning/milestones?direction=asc&sort=title&state=open).
 
+Getting Started
+---------------
+
+First and foremost, start small.
+
+This doesn't mean don't be ambitious with the breadth and depth of your contributions but rather
+understand the project context and culture before investing an asymmetric number of hours on
+development compared to your merged work.
+
+Even if you have an extensive open source background or sound software engineering skills, consider
+that comprehension by the other set of regular contributors is as much important than its technical
```

Comment:
It's ambiguous as to what "its" is referring to. Should it say _is_?

Reply:
Corrected, added code before comprehension.